devops/sources_backup.rst

```diff
@@ -0,0 +1,122 @@
+- Its URL can then be set in ``core.sources:download_urls`` and ``core.sources:upload_url``.
+- In your recipe's ``source()`` method, ensure the relevant ``get``/``download`` calls supply the ``sha256`` signature of the downloaded files.
+- Set ``core.sources:download_cache`` in your :ref:`global.conf<reference_config_files_global_conf>` file if a custom location is desired,
+  else the default cache folder will be used
+- Run Conan normally, creating packages etc.
+- Once some sources have been locally downloaded, the path pointed to by ``core.sources:download_cache`` will contain, for each downloaded file:
+   - A blob file (no extensions) with the name of the ``sha256`` signature provided in ``get``/``download``.
+   - A ``.json`` file which will also have the name of the ``sha256`` signature,
+     that will contain information about which references and which mirrors this blob belongs to.
+- Calling ``conan upload`` will now optionally upload the backups for the matching references if ``core.sources:upload_url`` is set.
+
+.. note::
+
+   Developers can also set this up in a similar way if needed, specially the ``core.sources:download_urls`` conf
+
+
+Creating the backup repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's create the backup repository with Artifactory CE. From the UI, create a generic repo to store the files, we'll imaginatively call it "source-backups".
+<IMAGE>
+
+Next, we want to allow anonymous read access for our backups
+(We'll touch on the :ref:`source_credentials.json<reference_config_files_source_credentials>` feature for restricting upload access,
+if you also want restricted read access, follow those same steps for reading). Create a new user, we'll call it "backup reader",
+and from the "Access" tab, give it read permissions to our "source-backups" repo.
+<IMAGE>
+**CHECK, anonymous read is done the other way around!**
+
+As for uploading permissions, we'll do the same now. Create a new user, this time "backup uploader", and give it "Manage" permisisons.
+<IMAGE>
+We can now create an access token for our "backup uploader" user and store it. This token needs to go into the `source_credentials.json` file
+of our machine/CI worker.
+<IMAGE>
+
+
+Setting up the necessary configs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The CI machine's :ref:`global.conf<reference_config_files_global_conf>` file should contain the
+``core.sources:download_urls`` and ``core.sources:upload_url`` items, and the `source_credentials.json` file
+should also contain the access token we generated in the previous step.
+<IMAGE>
+
+There are a few strategies possible in the ``download_urls`` conf depending on your use-case:
+
+* Origin first: ``["origin", "https://your.backup/remote/"]`` to look into and fetch from your backup remote only if and when the original source is not present.
+* Remote first: ``["https://your.backup/remote/", "origin"]`` to prefer your remote ahead of the recipes.
```

Comment:
I feel that ``confs`` have already been described like 3 times, maybe things can be collapsed a bit, it reads like a bit of duplication?